packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5199,7 +5308,6 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
     });
 });
 
-
```

Comment:
Usually between modules double line-breaks are used. It is not necessary to leave just one, especially since this change is not related to fixing the bug.